package.json

```diff
@@ -46,10 +39,22 @@
   },
   "devDependencies": {
```

Comment:
The `devDependencies` should be installed by default, except for in the case of:

> With the `--production` flag (or when the `NODE_ENV` environment variable is set to `production`), npm will not install modules listed in `devDependencies`.

&mdash; via https://docs.npmjs.com/cli/install#description

**TL;DR:** nope, should be installed by default unless you're doing `npm install --production` or `NODE_ENV=production npm install`.